test/all.js

```diff
@@ -37,6 +38,30 @@ test('can put/get two blobs in one core', async t => {
   t.end()
 })
 
+test("block size isn't affected by chunk size of streams", async (t) => {
+  const core = new Hypercore(ram)
+  const blockSize = 2 ** 16
+  const blobs = new Hyperblobs(core, { blockSize })
+
+  const buf = Buffer.alloc(5 * blockSize).fill('abcdefg')
+
+  // Write chunks to the stream that are smaller and larger than blockSize
+  for (const chunkSize of [blockSize / 2, blockSize * 2]) {
+    const ws = blobs.createWriteStream()
+    for (let i = 0; i < buf.length; i += chunkSize) {
+      const chunk = buf.slice(i, i + chunkSize)
+      ws.write(chunk)
+    }
+    ws.end()
+    await once(ws, 'finish')
+    const { blockOffset } = ws.id
+    const value = await core.get(blockOffset)
+    t.equal(value.length, blockSize)
```

Comment:
this should be t.equal(value.length, chunkSize), which also makes the test pass

Reply:
From my perspective this is not the expected behaviour. I would expect the size of the blocks that are written to the Hypercore to be the size specified by `options.blockSize` in the constructor. This test is showing that when using a write stream then the blocks can be either smaller or larger than `options.blockSize` - the size is dependent on the size of the chunks that are written to the write stream. I would expect the behaviour to be the same for both `hyperblobs.put()` and `hyperblobs.createWriteStream()` (`hyperblobs.put()` chunks the blob according to `options.blockSize`, `createWriteStream()` currently does not do this, which is what this test is trying to show).
Would you accept a PR to change this behaviour (e.g. make this test pass as-is, by ensuring that a blob written via `createWriteStream()` also is chunked according to `options.blockSize`)?